op-node/rollup/types.go

```diff
@@ -271,14 +269,6 @@ func (cfg *Config) Check() error {
 	if cfg.ChannelTimeoutBedrock == 0 {
 		return ErrMissingChannelTimeout
 	}
-	if cfg.GraniteTime != nil {
-		if cfg.ChannelTimeoutGranite == 0 {
-			return ErrMissingChannelTimeout
-		}
-		if cfg.ChannelTimeoutGranite > cfg.ChannelTimeoutBedrock {
```

Comment:
It's not a problem for derivation and FPs if any chain already uses a lesser `ChannelTimeoutBedrock`.